pkg/blockatlas/marketdata.go

```diff
@@ -0,0 +1,58 @@
+package blockatlas
+
```

Comment:
why this needs to be exposed in blockatlas package? 

it might be better to have everything inside `pricing` module

Reply:
because we going to use this model as API response